cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs

```diff
@@ -2162,7 +2162,45 @@ impl_runtime_apis! {
 				}
 
 				fn worst_case_asset_exchange() -> Result<(XcmAssets, XcmAssets), BenchmarkError> {
-					Err(BenchmarkError::Skip)
+					use xcm::latest::MAX_ITEMS_IN_ASSETS;
+
+					let relay_location = WestendLocation::get();
```

Comment:
We use `native_token` or `native_asset` representing relaychain token (`Location::new(1, Here)`) - e.g. ROC, WND, DOT, KSM, so I would suggest:
```suggestion
					let native_asset_location = WestendLocation::get();
					let native_asset_id = AssetId(native_asset_location.clone());
```
like this :point_up:, the naming will be aligned with this bellow:
```
						let asset_location = Location::new(1, [Parachain((2000 + i) as u32)]);
						let asset_id = AssetId(asset_location.clone());
```